### DISCUSSION TITLE
great Digital and one feature

### DISCUSSION THREAD

**Discussion by hycmos** [hneemann/Digital#854] [Ideas]
Digital is great.
I used the Logisim before. The Logisim can be downloaded from sourceforge. but Logisim has not been updated from 2013.
even not updated, There is still large downloads in SF every week, 17,815 This Week.
personally speaking, Digital is very good. It is more powerful.
Can Digital be placed on SF to find more users to expand its usage?

BTW, could one feature be added to the next version:
menu-->analysis-->expression-->create: 
can the circuit be simplified these boolean statements if the input is not simplified style?

Thanks.

**Comment by hneemann** [comment]
You can simply re-analyze the generated circuit. (menu->Analysis->Analysis)
Then the minimized boolean expression will be shown.
A new minimized circuit can then be created from the table view menu. (menu->create->Circuit)

**Reply by hycmos** [reply]
OK.
Thanks!
